mkdocs/docs/graphql/v2.0.0/api.html

```diff
@@ -189,6 +189,13 @@ <h5>Operations</h5>
               </li>
             </ul>
           </section>
+          <section>
+            <a
+            href="https://litmuschaos.github.io/litmus/auth/v2.0.0/api.html"
+            target="_blank"
+            >Authentication</a
+          >
+          </section>
```

Comment:
Could you please change
https://github.com/litmuschaos/litmus/blob/f801670b7327c16b9b80487e0e0c7ffd4ea82df7/mkdocs/docs/graphql/v2.0.0/api.html#L604
to
```
<td>Optional</td>
```